projects/varnish/project.yaml

```diff
@@ -1,7 +1,7 @@
 homepage: "https://varnish-cache.org/"
+language: c++
 primary_contact: "federico.schwindt@gmail.com"
 auto_ccs:
   - "phk@varnish.org"
-sanitizers:
-  - address
-  - undefined
+  - "p.antoine@catenacyber.fr"
```

Comment:
I think @bsdphk should be the primary contact with @nigoroll and @mbgrydeland as the sole auto CCs since they are currently our security contacts:

https://varnish-cache.org/security/index.html#i-have-found-a-security-hole

No offense meant, but there shouldn't be any other contact for oss-fuzz findings.

Reply:
Updated, let me know if it is good for you

Reply:
These mail addresses do not seem to be linked to a google account cf 
https://bugs.chromium.org/u/phk@varnish.org/
This will be needed to access the reports cf https://google.github.io/oss-fuzz/faq/#why-do-you-require-a-google-account-for-authentication